README.md

```diff
@@ -64,10 +64,8 @@ Please send us your contribution! See [the developers guide](https://github.com/
 
 ## External links
 
-* [ForensicsArtifacts.com ... the definitive database](http://forensicartifacts.com/)
 * [GRR Artifacts](https://www.blackhat.com/docs/us-14/materials/us-14-Castle-GRR-Find-All-The-Badness-Collect-All-The-Things-WP.pdf), by Greg Castle, Blackhat 2014
 
 ## Contact
 
-[forensicartifacts@googlegroups.com](https://groups.google.com/forum/#!forum/forensicartifacts)
-
+[slack](https://open-source-dfir.slack.com/messages/CBSJ9TDR9)
```

Comment:
forensicartifacts@googlegroups.com is still valid, undo remove

Reply:
Is it only Google internal? Because I get an access denied (#418) on that page.

Reply:
It's a private (external) Google group, you'll need to subscribe to it